soak/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.8-alpine
+FROM docker.io/library/python:3.8-alpine
```

Comment:
any reason this change was needed?

Reply:
While testing with docker run my script was stopping and asking for input on which python:3.9-alpine image to use? "docker.io, quay.io etc etc" ... so made the image path more specific and removing human interaction.

Reply:
Yeah I appreciate changing all tags to long form, especially as there are multiple public repositories (quay, docker, ECR public)